lib/krane/kubeclient_builder.rb

```diff
@@ -55,6 +55,14 @@ def build_policy_v1beta1_kubeclient(context)
       )
     end
 
+    def build_apps_v1_kubeclient(context)
+      build_kubeclient(
+        api_version: "v1",
+        context: context,
+        endpoint_path: "/apis/apps"
+      )
+    end
+
     def build_apps_v1beta1_kubeclient(context)
       build_kubeclient(
         api_version: "v1beta1",
```

Comment:
Do we need the apps_v1beta1 client anymore?

Reply:
Yeah, we still need it because of Ingress, which still uses `extensions/v1beta1`: 

https://github.com/Shopify/kubernetes-deploy/pull/598/files#diff-53358cd61c3052c22151ef2a24e68e9bL54 and https://github.com/Shopify/kubernetes-deploy/blob/fix_krane_k8s_api_1.16/test/fixtures/hello-cloud/web.yml.erb#L1.

Reply:
That's `extensions_v1beta1_kubeclient`, though, not `apps_v1beta1_kubeclient`, unless I'm missing something?

Reply:
Oh, you're absolutely right! I don't think we need `apps_v1beta1_kubeclient` anymore, would it be a problem to remove it in a subsequent PR?

Reply:
It's no problem, just trying to avoid cruft building up :)

Reply:
Just pushed a commit removing `apps_v1beta1_kubeclient` as it was a small change! Thanks for pointing it, Tim! ❤️